samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/Program.cs

```diff
@@ -33,9 +33,13 @@ private static int Main(string[] args)
         public void Dump()
         {
             // Get the blob reference
-            var storageAccount = CloudStorageAccount.Parse(StorageConnectionString);
-            var blobClient = storageAccount.CreateCloudBlobClient();
-            var blob = blobClient.GetBlobReferenceFromServer(new Uri(EventHubsCaptureAvroBlobUri));
+            string connectionString = "ConnectionString";
+            string containerName = "containerName";
+            string blobName = "blobName";
```

Comment:
There are constants that are defined for this purpose a few lines above. Can we not re-use them?
Instead of containerName and blobName, we should be able to use `EventHubsCaptureAvroBlobUri` by creating the BlobClient directly without needing to use the ContainerClient

https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.blobclient.-ctor?view=azure-dotnet

Reply:
I have update it to use `EventHubsCaptureAvroBlobUri`  creating the BlobClient directly.